noxfile.py

```diff
@@ -54,6 +57,7 @@ def tests_impl(
         "--strict-config",
         "--strict-markers",
         *(session.posargs or ("test/",)),
+        *pytest_extra_args,
```

Comment:
While a good idea, I'd like to be consistent: either we migrate integration and byte_string_comparisons to pytest_extra_args, or we name this emscripten_extra_args or pyiodide_extra_args.

Also it should be *before* `session.posargs` if possible to allow manual override of options using `nox -s  session -- posarg1 posarg2`.

Reply:
(I will do this later, let's focus on the important things)